quaistats/quaistats.go

```diff
@@ -844,12 +856,41 @@ func (s *Service) reportStats(conn *connWrapper) error {
 		sync := s.backend.Downloader().Progress()
 		syncing = header.Number().Uint64() >= sync.HighestBlock
 	}
+
+	cpuPercent, err := cpu.Percent(0, false)
+	if err != nil {
+		log.Warn("Error getting CPU usage:", err)
+		cpuPercent = []float64{-1.0}
+	}
+
+	// Get RAM usage
+	vmStat, err := mem.VirtualMemory()
```

Comment:
I'd also like to report absolute value of memory used. Eg if we're using 5GB of RAM, that would be something to potentially look into regardless of how much the system has available.

Reply:
CPU can stay the same. As for swap, I think we have enough with ram usage, but we could talk about it.